extensions/common/metadata_object.cc

```diff
@@ -231,7 +231,8 @@ flatbuffers::DetachedBuffer convertWorkloadMetadataToFlatNode(const WorkloadMeta
   node.add_labels(labels_offset);
   auto data = node.Finish();
   fbb.Finish(data);
-  return fbb.Release();
+  auto fb = fbb.Release();
+  return std::string(reinterpret_cast<const char*>(fb.data()), fb.size());
```

Comment:
nit, can you use bit_cast here instead?

Reply:
We're still in C++17 here, but we can switch to C++20 shortly.